websites/A/AnimesWatch/presence.ts

```diff
@@ -0,0 +1,59 @@
+const presence = new Presence({
+		clientId: "972647321970016316",
+	}),
+	strings = presence.getStrings({
+		playing: "general.playing",
+		paused: "general.paused",
+		browsing: "general.browsing",
+	}),
```

Comment:
Implement multiLang here too

Reply:
what

Reply:
You can find [docs](https://docs.premid.app) regarding this and use the repo to find examples

Reply:
I guess I didn't understand, is there a link that can be an example?

Reply:
* [metadata docs](https://docs.premid.app/en/dev/presence#filling-in-the-metadatajson-file)
  * See the first setting in the example metadata
* [`getStrings` docs](https://docs.premid.app/en/dev/presence/class#getstringsobject)
  * Read the examples
 
For a presence implementing multiLang, an example could be the Anime Planet presence